src/hotspot/share/classfile/vmSymbols.hpp

```diff
@@ -422,7 +422,7 @@ class SerializeClosure;
   template(sp_name,                                   "sp")                                       \
   template(pc_name,                                   "pc")                                       \
   template(cs_name,                                   "cs")                                       \
-  template(get_name,                                  "get")                                      \
```

Comment:
A bit surprised that this is not used elsewhere - but as long as this passes compilation I guess this is fine.